content/news/036/index.md

```diff
@@ -66,6 +67,28 @@ _Discussions:
 If needed, a section can be split into subsections with a "------" delimiter.
 -->
 
+## Announcements
+
+### [Ferris 3D got released!][ferris3d-github]
```

Comment:
I'm curious about this going in the announcements section, but I guess we've been using it quite loosely in the last few months. I think because people can use this in their games and it's quite general, it should be alright.

Reply:
I asked @ozkriff where to put it, and he suggested Announcements. I don't want to come off as trying to snatch the prime spot of the newsletter here, feel free to move it somewhere else if necessary 🙏.

Reply:
No worries, I'm happy with the decision if others have thought about it, just wanted to mention it :)

Reply:
Yeah, I think this specific section is a fine non-official announcement that doesn't fit any other section group too well anyway.